### DISCUSSION TITLE
Callback after loading the model

### DISCUSSION THREAD

**Discussion by farhadham** [splinetool/r3f-spline#7] [Q&A]
Hi everyone, My very first issue on GitHub, so I apologize if I'm doing anything wrong

How to know if the model is loaded from the URL and executes some functions afterward?
Is there any feature similar to onLoad?

**Comment by marcofugaro** [comment]
Hey, not sure what you're trying to do, but you should ask this question to the react-three-fiber community.

The behaviour should be the same of any react-three-fiber loader, for example [useLoader](https://docs.pmnd.rs/react-three-fiber/api/hooks#useloader).

**Reply by farhadham** [reply]
Basically, displaying a loading screen before the Canvas
I'm using the Suspense, but I assume the fallback is only used for the Canvas

Here is my demo
https://farhadf.com

**Comment by marcofugaro** [comment]
Yes, you can use the fallback prop to show a loading. It will show the loading until the scene is downloaded.
